api/v2/utilities/logs_validator.py

```diff
@@ -0,0 +1,36 @@
+from flask import request, jsonify, make_response, json
+from functools import wraps
+from cerberus import Validator
+
+
+def validate_log_data(func):
+    '''
+    This is a decorator to validate log data from the request
+    '''
+    @wraps(func)
+    def wrapper():
+        schema = {
+            "calendar_id" : {'type': 'integer', 'empty': False},
+            "subscriber_name" : {'type': 'string', 'empty': False },
+            "subscription_method" : {'type': 'string', 'empty': False },
+            "payload" : {'type': 'string', 'empty': False},
+        }
+        if func.__name__ == 'post':
+            data = request.get_json()
+            try:
+                check_data = {
```

Comment:
This would have looked cleaner if you made use of the data object.

Reply:
Yes, but this will allow users to provide their own field in the request. So this `check_data` gets specific fields in the request and an error will be thrown if one is missing